org.openttd.OpenTTD.appdata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Have you tried contributing this file upstream?

Reply:
They have been pinged, see Github link ^.